uwblueprint/community-fridge-kw
# feat: account info page

## Notion ticket link

[My Account Screen](https://www.notion.so/uwblueprintexecs/Team-Hub-5b0b225bfc09415ebcfad3030ce5fd30?p=fb252d0f22a347348ceae963feb6392e)



## Implementation description
* account information for donor (view only) 
* a bit different from the designs, went over changes with binal!



## Steps to test
1. log in 
2. navigate to my account page from nav bar
3. verify the correct information is shown



## What should reviewers focus on?
* 


## Checklist
